web/courses/views.py

```diff
@@ -253,6 +254,7 @@ def form_valid(self, form):
 class ProblemSetUpdate(UpdateView):
     model = ProblemSet
     fields = ["title", "description", "visible", "solution_visibility"]
+    pk_url_kwarg = "course_pk"
```

Comment:
A je to sigurno pravilno, a naj ne bi `pk_url_kwarg` povedal, pod katerim imenom se skriva PK od `ProblemSeta`, ki ga popravljamo?